src/main/java/com/mraof/minestuck/item/SburbCodeItem.java

```diff
@@ -0,0 +1,175 @@
+package com.mraof.minestuck.item;
+
+import com.mraof.minestuck.util.BlockHitResultUtil;
+import com.mraof.minestuck.util.MSTags;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.nbt.StringTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.item.context.UseOnContext;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import net.minecraft.world.phys.HitResult;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+/**
+ * Used for the Sburb Code item, extends ReadableSburbCodeItem which is used by Completed Sburb Code. Allows players to store a list of blocks from the GREEN_HIEROGLPYH block tag that in game reflects the collection of genetic code for the creation of sburb
+ */
+public class SburbCodeItem extends ReadableSburbCodeItem
```

Comment:
Odd naming convention with `SburbCodeItem` being a more specific version of `ReadableSburbCodeItem` instead of the other way around. I suggest renaming this class to `IncompleteSburbCodeItem` or `FillableSburbCodeItem`.

Reply:
Changed it to IncompleteSburbCodeItem, am considering simplifying the parent class to just SburbCodeItem for simplicity